doc/nrf/protocols/matter/getting_started/adding_clusters.rst

```diff
@@ -360,8 +371,8 @@ To check if the sensor device is working, complete the following steps:
      You can use the :ref:`matter_template_network_mode_onboarding` listed earlier on this page.
    * Send Matter commands.
 
-   At the end of this procedure, the LED indicating the state of the Matter device programmed with the sample starts flashing in the Short Flash Off state.
-   This indicates that the device is fully provisioned, but does not yet have full IPv6 network connectivity.
+   At the end of this procedure, the LED indicating the state of the Matter device programmed with the sample starts presenting the Solid On state.
+   This indicates that the device is fully provisioned, and has established a CASE session with the controller.
```

Comment:
```suggestion
   This indicates that the device is fully provisioned, and has been able / is able to establish a CASE session with the controller.
```
?

To me, the current sentence indicates that the session is still valid.

Reply:
Because the CASE session is valid,
If you will run pairing via the Chip-Tool interactive, the next commands doesn't need to establish CASE session.
the Chip-tool "command" mode works in the way that each command establish new CASE session even if previous has been established and "saved".

Reply:
Discussed F2F